bin/pytorch_inference/Main.cc

```diff
@@ -128,17 +128,21 @@ int main(int argc, char** argv) {
     std::string restoreFileName;
     bool isRestoreFileNamedPipe{false};
     std::string logFileName;
-    bool isLogFileNamedPipe{false};
+    std::string logProperties;
     ml::core_t::TTime namedPipeConnectTimeout{
         ml::core::CBlockingCallCancellingTimer::DEFAULT_TIMEOUT_SECONDS};
 
+    LOG_INFO(<< "Parsing command args");
```

Comment:
Since you've confirmed it works locally this should be removed.  In production this would go to the processes (hard to see) stderr, not to the ES log, as it's before the logging gets redirected.  This may cause some obscure problem one day.